src/body.rs

```diff
@@ -1,4 +1,4 @@
-use futures_lite::{io, prelude::*, ready};
+use futures_lite::{io, prelude::*, ready, AsyncRead as Read};
```

Comment:
We should probably keep calling this `AsyncRead`. I think that's a naming mistake we made with `async-std`.

```suggestion
use futures_lite::{io, prelude::*, ready, AsyncRead};
```